improver_tests/nbhood/recursive_filter/test_RecursiveFilter.py

```diff
@@ -65,6 +65,15 @@ class Test_RecursiveFilter(IrisTest):
 
     """Test class for the RecursiveFilter tests, setting up cubes."""
 
+    def _mean_points(self, points):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5863598)